packages/core/src/QueryApi.ts

```diff
@@ -40,6 +53,22 @@ export default interface QueryApi {
    */
   with(options: Partial<QueryOptions>): QueryApi
 
+  /**
+   * Creates a cold observable of the lines returned by the given query.
+   *
+   * @param query the query text in the format specifed in `QueryOptions.type`
+   * @param variables variables to add to the query, when `QueryOptions.type` is "flux"
+   */
+  lines(query: string, variables?: Variables): Observable<string>
+
+  /**
+   * Creates a cold observable of the rows returned by the given query.
+   *
+   * @param query the query text in the format specifed in `QueryOptions.type`
+   * @param variables variables to add to the query, when `QueryOptions.type` is "flux"
+   */
+  rows(query: string, variables?: Variables): Observable<Row>
```

Comment:
naming: My preference herein would be QueryParameters, since such a naming convention (parameter) is used in the majority of existing libraries that do queries. A variable (to me) means something that can change ... and to be semantically correct ... these are variable bindings.

Reply:
will change, had it called "Params" before :)
`Parameters` (shadows builtin TS type)
`Bindings`
`ParameterBindings`
`VariableBindings`

whats your preference?

Reply:
actually these are not bindings, because the passed values get serialized into a flux AST immediately. so changing a passed in value after the call, wouldnt affect the query.
guess i will go with `parameters: QueryParameters`